### ISSUE TITLE
App package missing description

### ISSUE THREAD

**Issue by neynah** [ocdtrekkie/draw#2]
EtherDraw has the description "Whiteboard" on the App Market but not in the package itself. Therefore, it doesn't display any description on the /apps page.

![image](https://cloud.githubusercontent.com/assets/855341/17038095/d781617a-4f48-11e6-9e1f-ece5a3ae6396.png)

**Comment by ocdtrekkie** [comment]
Huh. That's super interesting. Did they hardcode that into the market somewhere or I guess manually add it to the app index data?

As the case is, I investigated updating this app, and I couldn't get it running in my development environment. (I've also gotten a request via Twitter to update this app with the latest upstream code.) I bugged @paulproteus about it, but we never got it dealt with.

I'm happy to get this updated, but totally out of my depth on figuring out why I couldn't get it running with either the current or updated upstream code in dev.
